rust/otap-dataflow/crates/otap/src/otap_exporter/config.rs

```diff
@@ -0,0 +1,111 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+//! Configuration for the OTAP Exporter
+
+use crate::compression::CompressionMethod;
+use serde::{Deserialize, Deserializer};
+
+/// Configuration for the OTAP Exporter
+#[derive(Debug, Deserialize)]
+pub struct Config {
+    /// The grpc endpoint to which OTAP service requests will be sent
+    pub grpc_endpoint: String,
+
+    /// The type of compression to use for the gRPC messages. default = zstd.
+    /// The value "none" can be used to disable compression,
+    #[serde(
+        default = "default_compression_method",
+        deserialize_with = "deserialize_compression_method"
+    )]
+    pub compression_method: Option<CompressionMethod>,
+
+    /// Configuration for the arrow payloads
+    #[serde(default)]
+    pub arrow: ArrowConfig,
+}
+
+/// Configuration for the arrow payloads produced by the [`OtapExporter`]
+#[derive(Debug, Deserialize)]
+pub struct ArrowConfig {
+    /// Compression to use for IPC serialized payloads within the BatchArrowMessages
+    ///
+    /// This defaults to "zstd", wherein arrow IPC stream will be compressed with IPC
+    /// independently of whatever compression gRPC may have been configured. This is on by
+    /// default, achieving "double compression" because:
+    /// (a) relatively cheap in CPU terms
+    /// (b) minor compression benefit
+    /// (c) helps stay under gRPC request size limits
```

Comment:
This comment reuses the same justification for the double compression explained here, which I'm assuming is still valid
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/adbd65ed00ff447a96ce9bdce1320424e5bb420d/exporter/otelarrowexporter/internal/arrow/exporter.go#L54-L61